docs/index.html

```diff
@@ -204,12 +204,41 @@
       <h2>Introduction</h2>
 
       <p>
-        This specification is designed to enable global supply chain stakeholders to benefit from emerging technological web
-        standards like the Verifiable Credential (VC) Data Model, Linked Data (i.e. JSON-LD), and the Decentralized Identifier
-        (DID) Spec. This specification is relevant to prevalent supply chains but also allows for other use cases across a
-        variety of market segments. The corresponding data exchange schemas are modeled for Verifiable Credentials, which is a
-        standard for cryptographically signing business data. Schema semantics are precisely defined by use of Linked Data
-        which leverage established vocabularies such as schema.org, GS1, and UN/CEFACT.
+        This specification seeks to enable a path towards supply chain digitization, at its foundation addressing new threats surfaces 
+        of a digital society:
+      </p>
+
+      <p>
+        <cite>
+          The private sector must adapt to the continuously changing threat environment, ensure its products are built and operate 
+          securely, and partner with the Federal Government to foster a more secure cyberspace.  In the end, the trust we place in 
```

Comment:
Should still maintain our international focus, should not refer to "the federal government" or similar, should be framed in working with governments frame

Reply:
```suggestion
          securely, and partner with governments to foster a more secure cyberspace.  In the end, the trust we place in 
```